src/connectedk8s/azext_connectedk8s/troubleshoot_diagnoser_job_with_proxycert.yaml

```diff
@@ -0,0 +1,73 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: azure-arc-troubleshoot-sa
+  namespace: azure-arc
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: secret-reader
+  namespace: default
+rules:
+- apiGroups:
+  - ''
+  resources:
+  - secrets
+  verbs:
+  - get
+  - list
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: read-pods
+  namespace: default
```

Comment:
add a comment that this gets replaced with release namespace if at all it's different from 'default'

Reply:
I tried to but am not able to add the comments during runtime changes in the yaml.